api/db/repository/user.py

```diff
@@ -66,6 +68,11 @@ def update(_db: Session, user_id: int, user: UserUpdate):
             update_user_encoded["password"]
         )
 
+    if "is_active" in update_user_encoded and isinstance(
```

Comment:
`
if update_user_encoded.get("is_active") is not None:
`

Reply:
I fixed